cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -81,8 +81,13 @@ Feature: Users can authneticate with OIDC authenticator
   Scenario: Non-existing username in ID token is denied
     Given I get authorization code for username "not_in_conjur" and password "not_in_conjur"
     And I fetch an ID Token
+    And I save the amount of log lines into "bookmark_not_in_conjur"
```

Comment:
i understand that the bookmark name isn't really needed as long as we don't run tests in parallel. as this is less readabe and makes the reader think why this is here, i suggest creating a step "I save the amount of log lines into bookmark" which will call the step above with some fixed value

Reply:
Thanks, i refactored please review